components/content-service/pkg/layer/provider.go

```diff
@@ -282,9 +282,27 @@ func (s *Provider) GetContentLayerPVC(ctx context.Context, owner, workspaceID st
 	// At this point we've found neither a full-workspace-backup, nor a legacy backup.
 	// It's time to use the initializer.
 	if gis := initializer.GetSnapshot(); gis != nil {
```

Comment:
I have a question not related to this PR change in the content-service component.

What use case will go into the `initializer.GetSnapshot()`?

Reply:
None really. It is added here so that we can correctly handle the case when snapshot or prebuild is using volume snapshot and return a correct image layer for it, as we don't need to restore any actual content, as it is already present on the disk from volume snapshot.